recipes/ca-certificates/bld.bat

```diff
@@ -0,0 +1,9 @@
+:: Create the directory to hold the certificates.
+if not exist %LIBRARY_PREFIX%\ssl mkdir %LIBRARY_PREFIX%\ssl
+if errorlevel 1 exit 1
+
+:: Copy the certificates from certifi.
+copy /y %SP_DIR%\certifi\cacert.pem %LIBRARY_PREFIX%\ssl
+if errorlevel 1 exit 1
+copy /y %SP_DIR%\certifi\cert.pem %LIBRARY_PREFIX%\ssl
+if errorlevel 1 exit 1
```

Comment:
So, I just copied these twice on Windows. How should I link them (if I should do that at all), @msarahan?